lib/calculator.rb

```diff
@@ -0,0 +1,17 @@
+module Calculator
+
+  def subtotal
+    cost = 0
+    order.contents.uniq.each {|item| cost += order.number_of(item)*item.price}
```

Comment:
Surrounding space missing for operator '*'.<br>Space between { and | missing.<br>Space missing inside }.